runtime/gc_base/ScavengerForwardedHeader.cpp

```diff
@@ -100,6 +82,6 @@ MM_ScavengerForwardedHeader::didObjectGrowOnCopy()
 {
 	/* this only applies to forwarded objects */
 	Assert_MM_true(isForwardedPointer());
-	return (GROW_TAG == ((UDATA)_preserved.clazz & GROW_TAG));
+	return (GROW_TAG == ((UDATA)getPreservedClassAndTags() & GROW_TAG));
```

Comment:
getPreservedClassAndTags() already returns UDATA type, so that no extra cast (here and a few other spots) is needed

Reply:
Fixed.